### ISSUE TITLE
Add support for Free Apple development certificates

### ISSUE THREAD

**Issue by sufuk** [OlivierLDff/QtIosCMake#23]
Apple now allows Free Developer accounts to flash their apps to real iphones (app certificate will be valid for only 7 days) .
If I try to build I get ;

error: No account for team "123456789A". Add a new account in the Accounts preference pane or verify that your accounts have valid credentials. (in target 'MyApp' from project 'MyApp')
error: No profiles for 'MyApp.MyApp' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'MyApp.MyApp'. (in target 'MyApp' from project 'MyApp')

But after that if I launch the MyApp.xocdeproj with XCode and go to Signing & Capabilities and check "Automatically signing" and select Team as Personal Team it works.

It will be avesome if this will automated too

**Comment by OlivierLDff** [comment]
Interesting news.
I won't have time to work on that, but you can have a look around here:
https://github.com/OlivierLDff/QtIosCMake/blob/c5a541eddf25554c300f316c48a1021866b1b2a1/AddQtIosApp.cmake#L383-L392
If you find how to automate that, I think that would be interesting for lots of people, wanting to play around and test Qt for iOs development.
I can help you if you want to make a PR.

**Comment by sufuk** [comment]
> May be #6 is related. There was @OlivierLDff's idea to compare project "fixed by xcode" and original generated by cmake, but I received paid account , so this is become not important for me.

Yeah actually it is related, for some reason apple doesn't allow you to see your Team id directly :). But it can be learn from key chain explained above.

**Comment by OlivierLDff** [comment]
Would you consider doing a PR to add your explaination to Readme for the next person looking for its TEAM_ID ?
